target/linux/ramips/dts/DWR-116-A1.dts

```diff
@@ -0,0 +1,139 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,dwr-116-a1", "ralink,mt7620n-soc";
+	model = "D-link DWR-116 A1";
+
+	gpio-keys-polled {
+	compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		lan1 {
+			label = "dwr-116-a1:green:lan1";
+			gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+		};
+
+		lan2 {
+			label = "dwr-116-a1:green:lan2";
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+		};
+
+		lan3 {
+			label = "dwr-116-a1:green:lan3";
+			gpios = <&gpio2 2 GPIO_ACTIVE_LOW>;
+		};
+
+		lan4 {
+			label = "dwr-116-a1:green:lan4";
+			gpios = <&gpio2 3 GPIO_ACTIVE_LOW>;
+		};
+
+		/*wan {  /*WAN LED IS UART TX*/
```

Comment:
Drop this one and add a note to the commit message, that the wan led is using the uartlite pins and you prefer to have a serial console rather than a WAN LED.

Reply:
@mkresin , are there any (easy) way to reconfigure the pins at runtime?

Reply:
No, it isn'T possible to reconfigure pins during runtime.